js/ui/html_editor/modules/toolbar.js

```diff
@@ -190,7 +192,9 @@ if(Quill) {
                 insertColumnRight: getTableOperationHandler(this.quill, 'insertColumnRight'),
                 deleteColumn: getTableOperationHandler(this.quill, 'deleteColumn'),
                 deleteRow: getTableOperationHandler(this.quill, 'deleteRow'),
-                deleteTable: getTableOperationHandler(this.quill, 'deleteTable')
+                deleteTable: getTableOperationHandler(this.quill, 'deleteTable'),
+                cellProperties: () => { showCellPropertiesForm(this.editorInstance, $(this.quill.getModule('table').getTable()[2].domNode)); },
+                tableProperties: () => { showTablePropertiesForm(this.editorInstance, $(this.quill.getModule('table').getTable()[0].domNode)); }
```

Comment:
```suggestion
                cellProperties: () => {
                    const domNode = getTargetTableNode(this.quill, 'cell');
                    showCellPropertiesForm(this.editorInstance, $(domNode));
            },
                tableProperties: () => {
                    const domNode = getTargetTableNode(this.quill, 'table');
                    showTablePropertiesForm(this.editorInstance, $(domNode));
            }
```

Let's simplify it. Now it looks like ninja code :) 

```js
getTargetTableNode(quill, partName) {
    const [table, , cell] = this.quill.getModule('table').getTable();
    return tablePart === 'table' ? table,domNode : cell.domNode;
}
```